scala-package/core/src/main/scala/ml/dmlc/mxnet/optimizer/SGLD.scala

```diff
@@ -84,4 +82,12 @@ class SGLD(val learningRate: Float = 0.01f, val rescaleGradient: Float = 1.0f,
 
   // Dispose the state it created
   override def disposeState(state: AnyRef): Unit = {}
+
+  override def serializeState(state: AnyRef): Array[Byte] = {
+    null
```

Comment:
throw a UnsupportedOperationException temporarily just in case the user call it?